js-exercises/deep-copy-object/deepCopyObject.js

```diff
@@ -0,0 +1,5 @@
+const _ = require('lodash');
+
+const deepCopyObject = objToCopy => _.cloneDeep(objToCopy);
```

Comment:
Use of `lodash` or any other utility library is not allowed for simple JS exercises.